.github/workflows/bump-java-version.yml

```
@@ -1,19 +1,34 @@
-name: Stub GH action for devoping new workflows [STUB]
+
+name: UPDATE
+
 on:
   workflow_dispatch:
+    inputs:
+      logstash_version:
+        description: 'Logstash version'
```

Do you think adding an example might be useful here?

```suggestion
        description: 'Logstash version (i.e: 1.2.3)'
```